src/__test__/DetailsDisplay.test.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import { BrowserRouter as Router } from 'react-router-dom';
+import store from '../redux/configureStore';
+import DetailsDisplay from '../components/DetailsDisplay';
+
+describe('Render components', () => {
+  test('render Details page', () => {
+    const tree = renderer
+      .create(<Router><Provider store={store}><DetailsDisplay /></Provider></Router>)
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] This tests is currently failing. It will be nice to fix it.

![image](https://user-images.githubusercontent.com/51296741/152618976-4014a3b5-9eed-4c6a-842f-9e5eafc981ed.png)